src/client/apps/edit/components/content/sections/footer/index.tsx

```diff
@@ -51,15 +51,15 @@ export default connect(
   mapDispatchToProps
 )(SectionFooter)
 
-const FooterContainer = styled.div<{ layout?: string }>`
+const FooterContainer = styled.div.attrs<{ layout?: string }>({})`
```

Comment:
Noticed you switched from the newer styled component syntax here to the newer one. Any particular reason?

Reply:
The new style is failing in tests where types all appear as undefined, I think we need to update styled components